docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,152 @@
+# Troubleshooting the Honeycomb OpenTelemetry Distro for Java
+
+## Seeing gRPC spans labeled `export`
+
+If you are seeing gRPC spans with the Export function (honeycomb RPC),
+it is likely because the auto-instrumentation agent is instrumenting
+your manual SDK.
+Don't use the manual SDK tracer.
+
+## Missing custom spans
+
+If you are not seeing custom spans, added from in-code instrumentation,
+it is likely because you are using incompatible versions of the agent and SDK.
+
+## SDK Usage
```

Comment:
This is currently under Troubleshooting?

Reply:
that was where folks suggested should go as of a few weeks ago. I can move it into its own page instead.

Reply:
I'm gonna reorganize these pages a bit because it kinda bugs me now